### PR TITLE
New version: o2sh.onefetch version 2.14.1

### PR DESCRIPTION
**Pull request by o2sh**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/89835)

### CODE REVIEW

File: manifests/o/o2sh/onefetch/2.14.1/o2sh.onefetch.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+PackageIdentifier: o2sh.onefetch
+PackageVersion: 2.14.1
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+UpgradeBehavior: install
+ReleaseDate: 2022-11-27
+AppsAndFeaturesEntries:
+- DisplayName: onefetch version v2.13.2
+  DisplayVersion: v2.13.2
```
Comment (russellbanks) [inline manifests/o/o2sh/onefetch/2.14.1/o2sh.onefetch.installer.yaml:12]: These are the values that need updating

**Review by russellbanks — CHANGES_REQUESTED** [review]

**Review by russellbanks — APPROVED** [review]

**Comment by russellbanks** [comment]
I'm not able to test if the DisplayVersion contains a "v" right now. Does this version have that @o2sh?

**Comment by o2sh** [comment]
>I'm not able to test if the DisplayVersion contains a "v" right now. Does this version have that @o2sh?

No, I took the "v" out of the tag name. It's just 2.14.1 and not v2.14.1 --> https://github.com/o2sh/onefetch/releases/tag/2.14.1

File: manifests/o/o2sh/onefetch/2.14.1/o2sh.onefetch.installer.yaml (hunk 1 above)
Comment (o2sh) [reply manifests/o/o2sh/onefetch/2.14.1/o2sh.onefetch.installer.yaml]: Yes, I removed them as they're not needed anymore.

**Comment by russellbanks** [comment]
> >I'm not able to test if the DisplayVersion contains a "v" right now. Does this version have that @o2sh?
> 
> No, I took the "v" out of the tag name. It's just 2.14.1 and not v2.14.1 --> https://github.com/o2sh/onefetch/releases/tag/2.14.1

The DisplayName is the registry key shown in Control Panel

**Comment by o2sh** [comment]
Yes, if you use the installer linked to the release -> https://github.com/o2sh/onefetch/releases/download/2.14.1/onefetch-setup.exe

you will see this in the control panel:

image

File: manifests/o/o2sh/onefetch/2.14.1/o2sh.onefetch.installer.yaml (hunk 1 above)
Comment (russellbanks) [reply manifests/o/o2sh/onefetch/2.14.1/o2sh.onefetch.installer.yaml]: The ProductCode would still be worth keeping as it's the same through releases, so there wouldn't be any problem with automation. I can always add this at a later date though

**Pull request merged**